types/react-select/index.d.ts

```diff
@@ -485,7 +485,7 @@ declare namespace ReactSelectClass {
 declare class ReactSelectClass extends React.Component<ReactSelectClass.ReactSelectProps, ReactSelectClass.State> { }
 
 declare module ReactSelectClass {
-    class Creatable extends React.Component<ReactCreatableSelectProps, undefined> { }
+    class Creatable extends React.Component<ReactCreatableSelectProps, {}> { }
     class Async extends React.Component<ReactAsyncSelectProps, ReactSelectClass.AsyncState> { }
-    class AsyncCreatable extends React.Component<ReactAsyncSelectProps & ReactCreatableSelectProps, undefined> { }
+    class AsyncCreatable extends React.Component<ReactAsyncSelectProps & ReactCreatableSelectProps, {}> { }
```

Comment:
Tests are failing. I guess this should have `ReactSelectClass.AsyncState` as it's state.